vuepress/docs/next/tutorials/create/ms/spring-boot-ms.md

```diff
@@ -0,0 +1,117 @@
+```
+	Project=Maven
+	Language=Java
+	Spring Boot version=2.7.6
+	Group=com.entando.example
+	Artifact=spring-ms 
+	Name=spring-ms 
+	Description=Demo project for Spring Boot
+	Package name=com.entando.example.spring-ms
+	Packaging=Jar
+	Java=11
+	Dependencies= #under WEB: Spring Web 
+				  #under OPS: Spring Boot Actuator
+```
+
+2. Unzip the package and move the unzipped files and `src` directory to the `microservices/spring-ms/` directory.
+
+3. Create a new directory for the controller:
+```
+mkdir microservices/spring-ms/src/main/java/com/entando/example/springms/controller
+```
+4. In the controller directory, create `TemplateController.java` with the following content:
+``` java
+package com.entando.example.springms.controller;
+import org.springframework.web.bind.annotation.*;
+@RestController
+public class TemplateController {
+         @CrossOrigin @GetMapping("/api/example")
+         public MyResponse getExample() { return new MyResponse("test Data"); }
+         public static class MyResponse{
+                 private final String payload;
+                 public MyResponse(String payload) { this.payload = payload; }
+                 public String getPayload() { return payload; }
+         }
+}
+```
+
+5. To make the microservice compatible with Entando, add the following snippet to the `microservices/spring-ms/src/main/resources/application.properties` file:
+
+```
+server.port=8081
+management.endpoints.web.base-path=/api
+```
+6. Run the microservice locally to test that it works: 
+```
+ent bundle run spring-ms
+```
+
+- In your browser, access [http://localhost:8081/api/example](http://localhost:8081/api/example) to see `{"payload":"test Data"}`.
+- In your browser, access [http://localhost:8081/api/health](http://localhost:8081/api/health) to see `{"status":"UP"}`.
+In local development, the run command can be used to modify the port to run multiple microservices, but in production, microservices must run on port 8081.
+
+7. Create `microservices/spring-ms/Dockerfile` so ent knows how to assemble the Docker image for the service:
+
+```
+FROM openjdk:11
+WORKDIR /app
+COPY target/*.jar /app/app.jar
+CMD ["java", "-jar", "app.jar"]
+EXPOSE 8081
+```
+## Step 3: Build and Install the Bundle
+
+1. Optionally, add an image file in PNG or JPG format named thumbnail for the project.
+
+2. Package and install your bundle with the following commands:
+```
+ent bundle pack 
+ent bundle publish
+ent bundle deploy 
+ent bundle install 
+```
+3. To test the installed microservice, fetch your host name and microservice path with this command:
+``` 
+ent kubectl describe ingress 
+```
+- The URL where your microservice can be accessed follows this pattern:   
+`YOUR-HOST-NAME/YOUR-BUNDLE-NAME/YOUR-MS-NAME/`
+
+     With a local quickstart EntandoApp, the URL for this tutorial will look something like this:    
```

Comment:
why not just "For a quickstart application"? local and entandoapp are unnecessary and distracting